src/patternfly/components/DataList/data-list-text.hbs

```diff
@@ -1,6 +1,9 @@
-<span class="{{pfv}}data-list__text{{#if data-list-text--modifier}} {{data-list-text--modifier}}{{/if}}"
+<span class="{{pfv}}data-list__text{{#if data-list-text--modifier}} {{data-list-text--modifier}}{{/if}}{{#if data-list-text--IsTruncate}} pf-m-truncate{{/if}}"
   {{#if data-list-text--attribute}}
     {{{data-list-text--attribute}}}
-  {{/if}}>
+  {{/if}}>  
+  {{#ifAny data-list-text--IsTruncate}}
+    tabindex="0"
+  {{/ifAny}}
```

Comment:
The way I worded my suggestion in the last review implied you should use `ifAny` here, my bad. If it's just a single test, I'd use `if`

```suggestion
  {{#if data-list-text--IsTruncate}}
    tabindex="0"
  {{/if}}
```